app/config/variables.php

```diff
@@ -305,6 +305,46 @@
                 'required' => false,
                 'question' => '',
             ],
+            [
+                'name' => '_APP_STORAGE_DEVICE',
+                'description' => 'Select default storage device',
+                'introduction' => '0.7.0',
+                'default' => 'local',
+                'required' => false,
+                'question' => '',
+            ],
+            [
+                'name' => '_APP_STORAGE_DEVICE_S3_ACCESS_KEY',
+                'description' => 'AWS S3 storage access key',
```

Comment:
Maybe more info like: `AWS S3 storage access key. Used when default storage adapter is set to X. You can get this value from your AWS console.`

Reply:
Same for other S3* vars